lib/incoming_mail.rb

```diff
@@ -0,0 +1,110 @@
+module IncomingMail
```

Comment:
I feel like `lib` is becoming kind of a junk drawer for this codebase. Is there anywhere else it might make sense to put this?